test/AutoLayoutSpec.js

```diff
@@ -58,11 +48,6 @@ describe('bpmn-auto-layout', function() {
     });
 
 
-    it('parallel flows', function() {
```

Comment:
Removes this test scenario, cause it's body was empty

Reply:
:+1: